JunStack.java

```diff
@@ -200,67 +33,30 @@ public void push(String value) {
     public String pop(){
         if (trackStackPosition > 0) {
            String toPop = this.stack[trackStackPosition - 1];
-           System.out.println("Popping String " + toPop + " at index: " + (trackStackPosition - 1));
-           
+
            this.stack[trackStackPosition - 1] = null;
            trackStackPosition -= 1;
            
            return toPop; 
         } else {
-            System.out.println("Stack is empty.");
-            return "Stack is empty.";
+            return null;
         }
     }
     
     public boolean isEmpty(){
-        if (trackStackPosition == 0){
-            return true;
-        } else {
-            return false;
-        }
+       return trackStackPosition == 0;
     }
-    
-    public boolean isFull(){
-        if (trackStackPosition == stack.length){
-            return true;
-        } else {
-            return false;
-        }
-    }
-    
-    public String peek(){
+
+    public String peep(){
         if (trackStackPosition > 0){
             return this.stack[trackStackPosition - 1];
         } else {
-            return "Stack is empty.";
-        }
-    }
-    
-    public void change(int index, String replacement){

```

Comment:
The entire point of a stack is that you can only interact with the top of it, if you're editing the middle of it it's not a stack, it's a list

Reply:
fair